src/main/java/org/dinosaur/foodbowl/domain/photo/ImageUtils.java

```diff
@@ -0,0 +1,86 @@
+package org.dinosaur.foodbowl.domain.photo;
+
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import javax.imageio.ImageIO;
+import org.imgscalr.Scalr;
+import org.imgscalr.Scalr.Mode;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.multipart.MultipartFile;
+
+@Component
+public class ImageUtils {
+
+    @Value("${file.dir}")
+    private String fileDir;
+
+    public List<String> storeImageFiles(ImageType imageType, List<MultipartFile> files)
+        throws IOException {
+        List<String> storedImagesPaths = new ArrayList<>();
+
+        for (MultipartFile file : files) {
+            storedImagesPaths.add(storeImageFile(imageType, file));
+        }
+        return storedImagesPaths;
+    }
+
+    public String storeImageFile(ImageType imageType, MultipartFile file) throws IOException {
+        if (file.isEmpty()) {
+            throw new FileNotFoundException("파일이 존재하지 않습니다.");
+        }
+        String originalFilename = file.getOriginalFilename();
+        String storeFileName = createStoreFileName(originalFilename);
+        String path = imageType.getPath() + storeFileName;
+        String fullPath = getFullPath(path);
+
+        if (imageType.isResizable()) {
+            BufferedImage image = ImageIO.read(file.getInputStream());
+            image = resizeImage(image, imageType.getWidth(), imageType.getHeight());
+            ImageIO.write(image, extractExt(originalFilename), new File(fullPath));
+            return path;
+        }
+
+        file.transferTo(new File(fullPath));
+        return path;
+    }
+
+    private String createStoreFileName(String originalFilename) {
+        String uuid = UUID.randomUUID().toString();
+        String ext = extractExt(originalFilename);
```

Comment:
```suggestion
        String extention = extractExtention(originalFilename);
```
`ext`를 처음 봤을 때는 의미가 와닿지 않았습니다 🥲 명확하게 드러내는건 어떠신가요?

Reply:
파일의 확장자가 필요한 이유가 궁금합니다!

`jpeg`, `png` 중 하나를 정해두고 들어오는 모든 이미지는 `jpeg`, `png` 중 하나의 형식으로 저장되도록 하면 확장자를 알지 않아도 괜찮을 것 같다는 생각이 듭니다.

Reply:
위의 리뷰가 처리되지 않은 것 같아서 답변 부탁드립니다!

Reply:
> 파일의 확장자가 필요한 이유가 궁금합니다!
> 
> `jpeg`, `png` 중 하나를 정해두고 들어오는 모든 이미지는 `jpeg`, `png` 중 하나의 형식으로 저장되도록 하면 확장자를 알지 않아도 괜찮을 것 같다는 생각이 듭니다.

들어오는 이미지 파일들의 확장자를 검사하려면 확장자를 알아야하지 않을까요..?!